tests/modules/test_k8s_prerequisites_check.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+from unittest.mock import Mock
+
+from rookify.modules.exception import ModuleException
+from typing import Any, Dict, List
+from ..mock_k8s_prerequisite_check import MockK8sPrerequisitesCheckHandler
+
+
+# Note: currently this test works with pytest but not with unittest, which is not able to import needed classes
+class TestK8sPrerequisitesCheckHandler(unittest.TestCase):
+    def setUp(self) -> None:
+        # Mock configuration
+        self.config = {"rook": {"cluster": {"namespace": "test-namespace"}}}
+        # No response option
+        self.empty_response = False
+
+    def _request_callback(
+        self, method: str, *args: List[Any], **kwargs: Dict[Any, Any]
+    ) -> Any:
+        if method == "apps_v1_api.list_deployment_for_all_namespaces":
+            if self.empty_response is True:
+                return MockResponse([])
+            return MockResponse(["apple", "banana", "cherry"])
```

Comment:
I wonder if there's no equivalent available with a more intuitive name for `MockResponse`? See [NamespaceList](https://github.com/kubernetes-client/python/blob/041103952a93d1fa7a4478f972b581374f953ae6/kubernetes/docs/V1NamespaceList.md?plain=1). E.g. [DeploymentList](https://github.com/kubernetes-client/python/blob/041103952a93d1fa7a4478f972b581374f953ae6/kubernetes/client/models/v1_deployment_list.py)?

Reply:
ok changed name to 'V1DeploymentList'